src/widget.ts

```diff
@@ -45,11 +44,111 @@ export class SvgModel extends DOMWidgetModel {
 
 export class SvgView extends DOMWidgetView {
```

Comment:
Why do we define that view? It should probably be called SvgNadView if we keep it.